frontend/src/documents/Documents.svelte

```diff
@@ -43,6 +34,20 @@
       router.reload();
     }
   }
+
+  /**
+   * Rename the selected document with <F2>.
+   * Submit with <Enter> and exit with <Escape>.
+   */
+  function keyup(ev: KeyboardEvent) {
+    if (ev.key === "F2" && selected) {
+      moving = { ...selected, newName: basename(selected.filename) };
+    } else if (ev.key === "Enter" && moving) {
+      move();
+    } else if (ev.key === "Escape" && moving) {
```

Comment:
I think this should be solved slightly differently - the `AutocompleteInput.svelte` component that is used for the account input also has an event listener for `Escape` (and `Enter` too actually). So currently one can hit `Escape` in the account input to hide the list of suggestions. With this change, it would also close the dialog.

I think this handling of the Escape key could be done in the `ModalBase.svelte` component - then it would also benefit the "add transaction" dialog. There's already a keyboard shortcut handler in ModalBase for Escape, however that one is (like all the other "standard" keyboard shortcuts in Fava) not activated if you're currently in an input element or other editable element.

The following two changes would probably necessary: 1) add the keyboard handler in ModalBase 2) adjust the AutocompleteInput component to call `event.stopPropagation` if the suggestion list is hidden with the Escape key so that we do not inadvertently close both the list and the modal.